src/wp-admin/css/colors/_admin.scss

```diff
@@ -65,30 +66,6 @@ input[type=radio]:checked::before {
 	color: $link-focus;
 }
 
-input[type="text"]:focus,
```

Comment:
In this file, are these removed selectors dead code, or does the color scheme remove the need for them?

Reply:
Or maybe this is related to another feature or test - just wondering :)

Reply:
Since the CSS variables are used in the default stylesheets (common, form...) instead of hard-coded colors, all what's needed is to set the CSS variables properly for admin schemes like done line 11 in this file.

Reply:
Ohh I see - so these were a kind of mini-reset that is no longer required. Thanks for the clarification!